ui/client/config.js

```diff
@@ -15,7 +15,7 @@ let API_URL = `${nkPath}/api`
 const dateFormat = "YYYY-MM-DD HH:mm:ss"
 
 if (__DEV__) {
-  API_URL = 'http://localhost:8081/api'
+  API_URL = 'http://localhost:3000/api'
```

Comment:
can't we remove this "if" now - we'll be able to use relative path just like in production?

Reply:
Done